debian_package/sbin/folder2ram

```diff
@@ -310,14 +284,14 @@ Type=oneshot
 RemainAfterExit=yes
 ExecStart=/bin/true
 ExecStop=/sbin/folder2ram_cleaner
-TimeoutSec=30m
```

Comment:
Since this is a cleanup step, the 30 minutes is intentional. I want to make sure all data is synced to disk before tearing down everything.

Btw, afaik "30m" should be correct syntax here, see https://www.freedesktop.org/software/systemd/man/latest/systemd.service.html  "Takes a unit-less value in seconds, or a time span value such as "5min 20s". "

I'm not sure about the sleep command inside initscripts but I think it should be the same as sleep in console (it's the same binary /usr/bin/sleep from  coreutils where the help says "Pause for NUMBER seconds.  SUFFIX may be 's' for seconds (the default), 'm' for minutes, 'h' for hours or 'd' for days. " )